Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/clone.py

```diff
@@ -309,9 +309,11 @@ def pick_attr(name, plural, default=None):
                 sys.stderr.write('Completed clone, but failed to set parent, continuing...\n')
                 result += 1
 
+        if prefix:
+            raw_clone.title = '{} {}'.format(prefix, raw_issue.title)
+            raw_clone.commit_changes()
```

Comment:
Have you considered taking an optimistic approach – trying doing it all at one, and then step by step if it fails? Seems like it would be better for performance in the general case.

Reply:
This would be a lager change, I think. It will also take some work to make this change without duplicating the logic. I'll take a stab in a follow-up change.